onnxruntime/python/tools/quantization/quantize.py

```diff
@@ -605,41 +722,50 @@ def _get_quantize_input_nodes(self, node, input_index, qType):
         output_name = input_name + "_quantized"
 
         data_found, scale_name, zp_name, scale_shape, zp_shape = \
-                self._get_quantization_params(input_name)
+            self._get_quantization_params(input_name)
 
         if self.static:
             if data_found == False:
-                raise ValueError("Quantization parameters are not specified for param {}."
-                "In static mode quantization params for inputs and outputs of odes to be quantized are required.".format(input_name))
+                raise ValueError(
+                    "Quantization parameters are not specified for param {}."
+                    "In static mode quantization params for inputs and outputs of odes to be quantized are required."
+                    .format(input_name))
 
-            qlinear_node = onnx.helper.make_node("QuantizeLinear", [input_name, scale_name, zp_name], 
+            qlinear_node = onnx.helper.make_node(
+                "QuantizeLinear", [input_name, scale_name, zp_name],
                 [output_name], input_name + "_QuantizeLinear")
-            
             return [qlinear_node]
-            
+
         else:
             if data_found == True:
-                qlinear_node = onnx.helper.make_node("QuantizeLinear", [input_name, scale_name, zp_name], 
+                qlinear_node = onnx.helper.make_node(
+                    "QuantizeLinear", [input_name, scale_name, zp_name],
                     [output_name], input_name + "_QuantizeLinear")
+                return [qlinear_node]
```

Comment:
Add this missing return. Other code changes mainly are format and comment change.